examples/annotated-http-service/src/main/java/example/armeria/server/annotated/ReactiveService.java

```diff
@@ -0,0 +1,33 @@
+package example.armeria.server.annotated;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.server.annotation.Get;
+import com.linecorp.armeria.server.annotation.HttpResult;
+import com.linecorp.armeria.server.annotation.ProducesJson;
+
+import io.reactivex.rxjava3.core.Single;
+
+@ProducesJson
+public class ReactiveService {
```

Comment:
Temporary service for testing purposes. I still need to test `Mono`, but I want to figure out how to apply the converters.

Reply:
I see! I've just tested an API that returns `Mono` and it indeed seems like that's not implemented yet.